README.md

```diff
@@ -68,7 +68,7 @@ As the start script runs, it will launch the Consul web UI and the Couchbase web
 
 ```
 Touchbase cluster is launched!
-Try scaling it up by running: ./start scale
+Try scaling it up by running: ./start.sh scale
```

Comment:
Is the inconsistency with https://github.com/tgross/triton-touchbase/pull/5/files#r44927378 intentional?